docs/sources/alerting/set-up/configure-high-availability/_index.md

```diff
@@ -15,9 +15,9 @@ weight: 300
 
 # Enable alerting high availability
 
-You can enable alerting high availability support by updating the Grafana configuration file. On Kubernetes, you can enable alerting high availability by updating the Kubernetes container definition.
+You can enable alerting high availability support by updating the Grafana configuration file, additional steps are required to make this work if you run Grafana in a Kubernetes cluster. Both setups are described in this tutorial.
```

Comment:
```suggestion
There are two ways of enabling alerting high availability depending on whether you use Grafana or if you run Grafana in a Kubernetes cluster.
```

Reply:
Ok:) So:
You can enable alerting high availability support by updating the Grafana configuration file. If you run Grafana in a Kubernetes cluster, additional steps are required. Both options are described below.

Reply:
Sounds good!